viewmodels/src/commonMain/kotlin/com/mirego/trikot/viewmodels/PickerViewModel.kt

```diff
@@ -0,0 +1,26 @@
+package com.mirego.trikot.viewmodels
+
+import org.reactivestreams.Publisher
+
+interface PickerViewModel<T> {
+    /**
+     * List of elements in the spinner
+     */
+    val elements: Publisher<List<PickerItemViewModel<T>>>
+
+    /**
+     * Appearing value on the spinner
+     */
+    val selectedValueIndex: Publisher<Int>
```

Comment:
I would use the term `element` here instead of value. Or rename `elements` to `values` to keep consistency.